docs/management_api.yml

```diff
@@ -4,6 +4,7 @@ info:
   title: Device authentication
   description: |
       An API for device authentication handling.
+      This API version has been deprecated and will be removed in Mender 1.8 release.
```

Comment:
Admission needs to get also a note

Reply:
yes; I need to add a note to 1.6 admission api and remove admission api from master/1.7

Reply:
1.6 release contains admission 1.4, so is it ok to just add a note to admission 1.4? or do I need to create separate deviceadm branch - 1.6.x, add a note there and update integration 1.6 to contain admission 1.6?

Reply:
I think you mean [this one](https://github.com/mendersoftware/deviceadm/commit/54185dae47189d7db3c4bdd07bce8cb6f391de5a), right? I submitted it yesterday.